app/views/pages/terms_and_conditions.html.haml

```diff
@@ -0,0 +1,122 @@
+- @page_title = "Terms and conditions"
+= content_for :body_classes, 'terms-and-conditions-page'
+
+%nav.breadcrumbs{role: "Navigation", "aria-label" => "Breadcrumb"}
+  %ol{data: {"click-events" => "", "click-category" => "Header", "click-action" => "Breadcrumb Clicked"}}
+    %li.breadcrumbs__item
+      = link_to 'GOV.UK Registers', root_path
+    %li.breadcrumbs__item.breadcrumbs__item--active
+      = link_to 'Terms and conditions', '#content', "aria-current" => "page"
+
+.govuk-width-container
+  %main#content{class: 'govuk-main-wrapper govuk-!-padding-bottom-9', role:'main'}
+    .govuk-grid-row
+      %section.govuk-grid-column-two-thirds#content
+        %h2.govuk-heading-l Terms and conditions
+        %p This page and any pages it links to explains the terms of use of GOV.UK Registers. You must agree to these to use GOV.UK Registers.
+
+        %h2.govuk-heading-m Who we are
+        %p GOV.UK Registers is managed by #{link_to 'Government Digital Service', 'https://www.gov.uk/government/organisations/government-digital-service/'} (GDS) on behalf of the Crown. GDS is part of the Cabinet Office and will be referred to as 'we' from now on
+
+        %h2.govuk-heading-m Using GOV.UK Registers
+        %p You agree to use GOV.UK Registers only for lawful purposes. You must also use it in a way that does not infringe the rights of, or restrict or inhibit the use and enjoyment of, this site by anyone else.
+        %p We update GOV.UK Registers all the time. We can change or remove content at any time without notice.
```

Comment:
Is this talking about the website? If this refers to data then this would definitely put me off using the service